### ISSUE TITLE
Extra furnaces for Steam Heater

### ISSUE THREAD

**Issue by elifoster** [Flaxbeard/Flaxbeards-Steam-Power#246]
#113 

Some extra furnaces should be able to be heated by the steam heater. These should include, but are not limited to:

* [ ] Natura Nether Furnace
* [ ] IndustrialCraft 2 Iron Furnace
* [ ] Flaxbeard's Steam Power Crucibles

**Comment by Flaxbeard** [comment]
Should it heat crucibles for free as long as steam is in the system?

**Comment by elifoster** [comment]
No. It uses the same amount of steam as it does for regular furnaces; 20.

**Comment by Flaxbeard** [comment]
...why? The crucible doesn't use power normally

**Comment by elifoster** [comment]
Convenience fee. It's more convenient for builds to use a steam heater than lava + a bunch of blocks to contain it.

**Comment by Flaxbeard** [comment]
That seems like an excuse not to add 3 lines of code to disable cost on a crucible than an actual reason. Isn't spending brass+copper and hooking it up via pipes cost enough?

**Comment by elifoster** [comment]
That doesn't even make sense. It takes more code to make it cost than not.

**Comment by elifoster** [comment]
Here's a real world example to explain why I did it: When you cook food on a stovetop, it requires gas/electricity. It doesn't just require gas/electricity to initially turn it on, you need the gas/electricity to keep it hot. The Steam Heater shouldn't just be infinitely hot when there's a crucible on top of it.

**Comment by Gunner76th** [comment]
I would like to have the Tinker's Construct Slab Furnace added to this list please.
